### PR TITLE
[BE] correct coupon validation in admin

### PR DESCRIPTION
**Pull request by poxip**

### CODE REVIEW

File: aa_stripe/forms.py [hunk 1]
```diff
@@ -5,6 +5,14 @@
 
 
 class StripeCouponForm(forms.ModelForm):
+    def clean_currency(self):
```
Comment (jacoor) [inline aa_stripe/forms.py]: Please remove this, and simply set this not blank in admin. Every coupon requires currency.

Comment (poxip) [reply aa_stripe/forms.py]: If "percent_off" is specified, currency is not needed, but as we have "default" and there's a list of choices I can make this required in admin, this seems okay.

Comment (poxip) [reply aa_stripe/forms.py]: Added in the CR commit

**Comment by jacoor** [comment]
Right, good one. Did you check it on stripe to create couponts with this
setting required?

**Comment by poxip** [comment]
Unfortunately currency needs "blank=True, null=True", because if amount_off is not specified on Stripe, currency can be null. I will revert to have the validation in the admin form back.

**Comment by jacoor** [comment]
Great that you set nulls and blank, however  default there wil be very annoying and misleading for the user. I am removing it.

**Pull request merged**